examples/customresourceinterpreter/apis/workload.example.io_workloads.yaml

```diff
@@ -7698,7 +7698,7 @@ spec:
                                 user:
                                   description: |-
                                     user to map volume access to
-                                    Defaults to serivceaccount user
```

Comment:
Seems this comes from Kubernetes, and we can't fix it on our side.

Reply:
thanks